From dad94437ba64d93e00f19038c7c58a88ee448e7c Mon Sep 17 00:00:00 2001
From: Peter Vary <pvary@cloudera.com>
Date: Fri, 12 May 2017 12:21:30 +0200
Subject: [PATCH 1107/1164] CDH-53676 Hive Query return duplicate log in
 beeline           Caused by CDH-52465:
 HIVE-16146: If possible find a better way to
 filter the TestBeeLineDriver output(Peter Vary
 via Zoltan Haindrich, reviewed by Vihang
 Karajgaonkar)           Merge was unsuccessfull.
 This commit removes the extra line.

Change-Id: Iffe845c00c0e1275e8dabf927ab5b64844adbb7e
---
 .../src/java/org/apache/hive/beeline/Commands.java |    1 -
 1 file changed, 1 deletion(-)

diff --git a/beeline/src/java/org/apache/hive/beeline/Commands.java b/beeline/src/java/org/apache/hive/beeline/Commands.java
index 290c888..612bda5 100644
--- a/beeline/src/java/org/apache/hive/beeline/Commands.java
+++ b/beeline/src/java/org/apache/hive/beeline/Commands.java
@@ -1227,7 +1227,6 @@ public void run() {
                   // In test mode print the logs to the output
                   beeLine.output(log);
                 }
-                beeLine.info(log);
               }
               Thread.sleep(DEFAULT_QUERY_PROGRESS_INTERVAL);
             } catch (SQLException e) {
-- 
1.7.9.5

